doc/readme-zephyr: (#1486)
mcu-tools/mcuboot: docs/readme-zephyr.md

partitions defined in its device tree. These partitions are:

- `boot_partition`: for MCUboot itself
- `slot0_partition`: the primary slot of Image 0
- `slot1_partition`: the secondary slot of Image 0
- `scratch_partition`: the scratch slot

Currently, the two image slots must be contiguous. If you are running
MCUboot as your stage 1 bootloader, `boot_partition` must be configured
so your SoC runs it out of reset. If there are multiple updateable images
then the corresponding primary and secondary partitions must be defined for

> the rest of the images too (e.g. `slot2_partition` and

Reviewer: ```suggestion
the rest of the images too (for example, `slot2_partition` and
```

The generated keypair above contains both the public and the private
key.  It is necessary to extract the public key and insert it into the

> bootloader.  ``CONFIG_BOOT_SIGNATURE_KEY_FILE`` Kconfig option shall be used to

Reviewer: ```suggestion
bootloader.  Use the ``CONFIG_BOOT_SIGNATURE_KEY_FILE`` Kconfig option to
```

> provide the path to the key file so the build system can extracting
> the public key to form which is usable by the C compiler.
> The public key is generated to the `build/zephyr/autogen-pubkey.h` which is included
> by the `boot/zephyr/keys.c`.

Reviewer: ```suggestion
provide the path to the key file so the build system can extract
the public key in a format usable by the C compiler.
The generated public key is saved in `build/zephyr/autogen-pubkey.h`, which is included
by the `boot/zephyr/keys.c`.
```

> Currently zephyr-rtos port limits its support to one keypair at the time,
> despite MCUboot's key managment infrastructure allows multiple keypairs.

Reviewer: ```suggestion
Currently, the Zephyr RTOS port limits its support to one keypair at the time,
even if MCUboot's key management infrastructure supports multiple keypairs.
```

Author: Not `even if` because MCUboot's infrastructure does support multiple key.
